docs/shared/mental-model.md

```diff
@@ -116,7 +116,7 @@ execution time.
 
 ## Affected commands
 
-When you run `nx test app1`, you are telling Nx to run the `app1:test` task plus all the tasks it depends on.
+When you run `nx test app1`, you are telling Nx to run the `app1:test` task plus all the tasks they depend on.
```

Comment:
This should stay `it depends`.

There is a single task depending on the libraries.